index.js

```diff
@@ -0,0 +1,3 @@
+export function requireContract(contractName) {
```

Comment:
Just to reiterate what you already know:  The arc json contracts files are not supposed to contain migration addresses.  I think you view this as potentially just a work-around.  Just a heads up that a couple things, described in other comments here, don't sync with what I'm doing in arc-js (https://github.com/daostack/arc-js/issues/17), which is currently blocked by #226.  Hopefully that will be resolved very soon.